# all Linux versions don't have pthread_np.h, so just hardcode the single function prototype we'd be getting from there:

diff --git a/vendor/tcl8.5.19/unix/tclUnixPort.h b/vendor/tcl8.5.19/unix/tclUnixPort.h
index 965014eb38d63e890bd7d90461cd571f08f5a51d..ccf50f2110c17e587d09245d2b0418e9ef6610f2 100644
--- a/vendor/tcl8.5.19/unix/tclUnixPort.h
+++ b/vendor/tcl8.5.19/unix/tclUnixPort.h
@@ -631,7 +631,8 @@ typedef int socklen_t;
  * Assume it is in pthread_np.h if it isn't in pthread.h. [Bug 1064882]
  * We might need to revisit this in the future. :^(
  */
-#	    include <pthread_np.h>
+// #	    include <pthread_np.h>
+extern int pthread_attr_get_np(pthread_t thread, pthread_attr_t *attr);
 #	endif
 #   else
 #	ifdef HAVE_PTHREAD_GETATTR_NP
